src/Wellcome.Dds/Wellcome.Dds.AssetDomainRepositories/Mets/NamedAmazonS3ClientFactory.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System.Collections.Generic;
+using Amazon.S3;
+
+namespace Wellcome.Dds.AssetDomainRepositories.Mets
+{
+    /// <summary>
+    /// Support managing named <see cref="IAmazonS3"/> clients.
+    /// </summary>
+    /// <remarks>Note - as it stands this will only support Singletons</remarks>
+    public class NamedAmazonS3ClientFactory : INamedAmazonS3ClientFactory

```

Comment:
I think this class should live in Utils.Aws or somesuch, rather than here, as it will be used in places this assembly is not.

Reply:
Agreed - will move.